src/blue-theme.scss

```diff
@@ -13,6 +13,10 @@ $primary: $democracy-blue;
 @import "../node_modules/bulma-switch/dist/css/bulma-switch.sass";
 
 
+html {
+    overflow-y: auto;
```

Comment:
How sure are we that this won't have side-effects in the appearance of various segments? Did we test it properly after this was added? A "html" element has to be overwritten anywhere it goes. It might be better to define this behaviour in a class and apply it.